### PR TITLE
Fix issue when undefined data in AVM client caused FJS to fail

### PR DESCRIPTION
**Pull request by coder11**

### CODE REVIEW

File: avm/client/src/avmHelpers.ts [hunk 1]
```diff
@@ -162,6 +162,9 @@ export async function callAvm(
         return {
             retCode: -1,
             errorMessage: 'marine-js call failed, ' + e,
-        } as any;
+            data: new Uint8Array(),
+            nextPeerPks: [],
+            callRequests: [],
+        };
```
Comment (folex) [inline avm/client/src/avmHelpers.ts:168]: Now any errors from AVM call will be silently ignored? Or are they reported somewhere else?

Comment (coder11) [reply avm/client/src/avmHelpers.ts:168]: No, fjs will just not fail trying to parse undefined data

File: avm/client/src/avmHelpers.ts [hunk 2]
```diff
@@ -162,6 +162,9 @@ export async function callAvm(
         return {
             retCode: -1,
             errorMessage: 'marine-js call failed, ' + e,
-        } as any;
+            data: new Uint8Array(),
```
Comment (mikevoronov) [inline avm/client/src/avmHelpers.ts:165]: Can we make a separate result for marine-js, wrapping marine-js into AquaVM error looks wierd.

Comment (coder11) [reply avm/client/src/avmHelpers.ts]: Let's just do not handle errors on this level at all. The consumer should be responsible to handle errors in the `fn` passed into arguments

File: avm/client/src/avmHelpers.ts (hunk 1 above)
Comment (coder11) [reply avm/client/src/avmHelpers.ts]: I've removed this block whatsoever. See my answer to Mike

**Review by mikevoronov — APPROVED** [review]
utACK

**Pull request merged**